src/utils/metadata.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2021 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+import { getSetting } from 'docc-render/utils/theme-settings';
+import { normalizeAssetUrl } from 'docc-render/utils/assets';
+
+const themeTitle = getSetting(['meta', 'title'], process.env.VUE_APP_TITLE);
+const createMetaTags = ({ title, description, path }) => [
+  {
+    name: 'description',
+    content: description,
+  },
+  {
+    property: 'og:locale',
+    content: 'en_US',
+  },
+  {
+    property: 'og:site_name',
+    content: themeTitle,
+  },
+  {
+    property: 'og:type',
+    content: 'website',
+  },
+  {
+    property: 'og:title',
+    content: title,
+  },
+  {
+    property: 'og:description',
+    content: description,
+  },
+  {
+    property: 'og:url',
+    content: normalizeAssetUrl(path),
```

Comment:
I think we're now correctly accounting for a base path when it has been configured (thanks!), but I noticed that we are now creating metadata with relative paths as opposed to absolute URLs, and it seems that the open graph protocol specifies that URLs should be full/absolute URLs with scheme/host/etc.

I went ahead and created a PR to turn these relative paths into full/absolute URLs: https://github.com/marinaaisa/swift-docc-render/pull/3

Let me know what you think!